<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matcher</class>
 <widget class="QWidget" name="Matcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>匹配</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QGraphicsView" name="graphicsView_D"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>310</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>相机参数</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="maximumSize">
      <size>
       <width>310</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>匹配参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="6">
       <widget class="QLabel" name="label_40">
        <property name="minimumSize">
         <size>
          <width>316</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>316</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>uniquenessRatio(视差唯一性百分比)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_uniradio">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_uniradio">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>preFilterCap(前置滤波器截断值)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_prefilcap">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_prefilcap">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QLabel" name="label_36">
        <property name="minimumSize">
         <size>
          <width>316</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>316</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>SpeckleWindowSize(视差连通区域变化度的窗口大小)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_specwinsz">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="QLabel" name="label_37">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>SADWindowSize(SAD窗口大小)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="lineEdit_SadWinSiz">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_SadWinSiz">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>51</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>minDisparity(最小视差值)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="lineEdit_MinDisp">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_MinDisp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="minimum">
         <number>-64</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="5">
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>numDisparities(视差窗口大小)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="lineEdit_NumDisparities">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>16</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_NumDisparities">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="5">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>speckleRange(连通区域内的最大视差变化)</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="lineEdit_specrange">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_specrange">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>64</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="6">
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>disp12MaxDiff(左右视差图的最大容许差异)</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLineEdit" name="lineEdit_maxdifdisp12">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_maxdifdisp12">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="6">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>TextureThreshold(低纹理区域的判断阈值)</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLineEdit" name="lineEdit_textThread">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_textThread">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="18" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>Function</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_SGBM">
           <property name="text">
            <string>SGBM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_BM">
           <property name="text">
            <string>BM</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="18" column="3" colspan="3">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>MODE</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_MODE_HH">
           <property name="text">
            <string>HH</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_MODE_3WAY">
           <property name="text">
            <string>3WAY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_MODE_SGBM">
           <property name="text">
            <string>SGBM</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_MatchDefault">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>默认参数</string>
        </property>
       </widget>
      </item>
      <item row="19" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_MatchChooseFiles">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>双目匹配</string>
        </property>
       </widget>
      </item>
      <item row="19" column="4">
       <widget class="QPushButton" name="pushButton_RefreshStereoMatch">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="19" column="5">
       <widget class="QPushButton" name="pushButton_RangeSinglePoint">
        <property name="text">
         <string>测量</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QSlider" name="horizontalSlider_specwinsz">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
